community/kernel/src/main/java/org/neo4j/kernel/impl/newapi/NodeSchemaMatcher.java

```diff
@@ -35,11 +35,9 @@
  */
 public class NodeSchemaMatcher
 {
-    private final Read read;
 
     public NodeSchemaMatcher( Read read )
```

Comment:
if this class does not have any state now, should we have a possibility to create one? or static helper method should be enough?